projects/demo/src/modules/app/utils/example-content-processor/index.ts

```diff
@@ -0,0 +1,27 @@
+import {identity} from '@taiga-ui/cdk';
+import {FrontEndExample} from '../../../interfaces/front-end-example';
+import {isLess, isTS} from '../index';
+import {processLess} from './less-processor';
+import {processTs} from './typescript-processor';
+
+function getProcessor(fileName: string): (item: string) => string {
+    switch (true) {
```

Comment:
why don't you want to use standard `if-conditions`? ))

```ts
function getProcessor(fileName: string): (item: string) => string {
  if (isTS(fileName)) {
    return processTs
  } else if (isLess(fileName)) {
     return processLess;
  } else {
     return identity;
  }
}
```

Reply:
I decided that it is more readable way)

But if there is doubts about it, I will change it to your proposal.

Reply:
`switch (true) {` I was just confused by this)) but it's up to you

Reply:
fixed